cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.contains('button', 'Customer Login').click();
+    cy.get('#userSelect').select('Hermoine Granger');
+    cy.get('.btn-default').should('be.visible').click();
+
+    cy.contains('.center', 'Account Number')
+      .should('contain.text', 1001);
+    cy.contains('.center', 'Balance').should('contain.text', 5096);
+    cy.contains('.center', 'Currency').should('contain.text', 'Dollar');
+
+    cy.get('[ng-class="btnClass2"]').click();
+    cy.get('.form-control').type(1234);
+    cy.get('[type="submit"]').click();
+    cy.contains('[ng-show="message"]', 'Deposit Successful')
+      .should('be.visible');
+    cy.contains('.center', 'Balance').should('contain.text', 6330);
```

Comment:
The balance is hardcoded as 6330 after the deposit. Ensure this value is consistent with the test data or consider parameterizing it if it might change.